### PR TITLE
[PFC WD]: initial support

### PR DESCRIPTION
**Pull request by marian-pritsak**
**What I did**
Provided initial support for PFC watchdog.
**Why I did it**
Implemented PFC watchdog class derived from Orch. It is subscribed to PFC_WD_TABLE and based on configuration provided there runs watchdog on lossless queues.
**How I verified it**
Generated PFC storm on fanout switch, tried to send traffic on that port to see if it is properly forwarded/dropped.

### CODE REVIEW

**Comment by lguohan** [comment]
retest this please

**Comment by stcheng** [comment]
the test will fail because the swsscommon hasn't been merged yet. i'll merge the swsscommon one first.

**Comment by marian-pritsak** [comment]
retest this please

**Comment by marian-pritsak** [comment]
Depends on https://github.com/Azure/sonic-sairedis/pull/227

**Comment by stcheng** [comment]
the original commit comes with bad format commit messages. i've force-pushed this commit with a clearer message.

File: orchagent/orchdaemon.cpp [hunk 1]
```diff
@@ -37,6 +37,8 @@ bool OrchDaemon::init()
 {
     SWSS_LOG_ENTER();
 
+    string platform = getenv("platform") ? getenv("platform") : "";
```
Comment (qiluo-msft) [inline orchagent/orchdaemon.cpp]: >getenv [](start = 22, length = 6)

Could you getenv once?
There are other similar problems in this repo, please also check.

**Pull request merged**